### PR TITLE
Added AccountLinking webhook support

### PR DESCRIPTION
**Pull request by dmitryshagin**
Added AccountLinking callback and fixed :read callback.

### CODE REVIEW

File: lib/facebook/messenger/incoming/account_linking.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        end
+
+        def status
+          @messaging['account_linking']['status']
+        end
+
+        def authorization_code
+          @messaging['account_linking']['authorization_code']
+        end
+
```
Comment (nashby) [inline lib/facebook/messenger/incoming/account_linking.rb]: could you please remove extra blank lines here?

File: lib/facebook/messenger/bot.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@ module Bot
 
       base_uri 'https://graph.facebook.com/v2.6/me'
 
-      EVENTS = [:message, :delivery, :postback, :optin].freeze
+      EVENTS = [:message, :delivery, :postback, :optin, :read, :account_linking].freeze
```
Comment (nashby) [inline lib/facebook/messenger/bot.rb]: I just merged PR that fixes `:read` event. Could you please rebase your branch?

File: lib/facebook/messenger/incoming/account_linking.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+module Facebook
+  module Messenger
+    module Incoming
+      # The Optin class represents an incoming Facebook Messenger optin,
+      # which occurs when a user engages by using the Send-to-Messenger Plugin.
+      #
+      # https://developers.facebook.com/docs/messenger-platform/plugin-reference
```
Comment (sorich87) [inline lib/facebook/messenger/incoming/account_linking.rb]: You forgot to adapt the description.

**Comment by dmitryshagin** [comment]
Travis is now passing. Seems to be done.

File: lib/facebook/messenger/bot.rb (hunk 2 above)
Comment (dmitryshagin) [inline lib/facebook/messenger/bot.rb]: Already done. Should I do domething more to get merged?

**Comment by sorich87** [comment]
Thanks!

**Pull request merged**